terraform/environment/region/api_ecs.tf

```
@@ -475,6 +503,10 @@ locals {
         {
           name      = "ONE_LOGIN_CLIENT_ID"
           valueFrom = data.aws_secretsmanager_secret.gov_uk_onelogin_client_id.arn
+        },
+        {
+          name  = "LPA_STORE_JWT_KEY_SECRETMANAGER_ARN",
```

```suggestion
          name  = "LPA_STORE_JWT_KEY_SECRET",
```

If it's going to contain the secret probably this?

---

That being said we probably want to access the secret from the shared location for rotation purposes so maybe this shouldn't be a "secrets" value and should be an "environment" one with a name like "LPA_STORE_JWT_SECRET_LOCATION" (it's not quite a name since it's in a different account?)